pkg/gatewayserver/io/ws/lbslns/time.go

```diff
@@ -128,11 +128,13 @@ func TimePtrToUpInfoTime(t *time.Time) (float64, int64) {
 // ConcentratorTimeToXTime contructs the XTime associated with the provided
 // session ID and concentrator timestamp.
 func ConcentratorTimeToXTime(id int32, t scheduling.ConcentratorTime) int64 {
-	return int64(id)<<48 | (int64(t) / int64(time.Microsecond) & 0xFFFFFFFFFF)
+	return int64(id)<<48 | (int64(t) / int64(time.Microsecond) & 0xFFFFFFFFFFFF)
```

Comment:
This makes the `xtime` now correctly have upto 48 bits of timestamp.

Reply:
Ouch, ouch, ouch.

This happens in multiple places - please patch those (either now or in a separate PR) too.
https://github.com/TheThingsIndustries/lorawan-stack/search?q=0xFFFFFFFFFF

Reply:
I've fixed the other one here. I'll fix Tabs Hubs when I work on the merge PR.

Reply:
Damn this is a nasty one indeed. So after 12 hours of connectivity downlink breaks ...